executor/executor_xapi.go

```diff
@@ -332,7 +361,11 @@ func (e *XSelectIndexExec) doIndexRequest() (*xapi.SelectResult, error) {
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
-	return xapi.Select(txn.GetClient(), selIdxReq, 1)
+	concurrency := 1
```

Comment:
Why set this to one? The underlying tikv client will ensure the order when using multiple threads.